docs/docs/types/message.md

```diff
@@ -0,0 +1,48 @@
+Focused on deliverying outbound communication. Perhaps announcing a product launch, new content, top-level alert, etc. Message modules are simple text modules which can be used in a variety of formats.
+
+``` javascript
+var module = pathfora.Message({
+	// module settings
+});
+
+pathfora.initializeWidgets([ module ]);
+```
+
+
+## layout
+
+Define which layout type the form module should use.
+
+| Value | Type | Behavior |
+|---|---|---|
+| modal | string | module uses a [modal layout](../layouts/modal.md) |
+| slideout | string | module uses a [slideout layout](../layouts/slideout.md) |
+| bar | string | module uses a [bar layout](../layouts/bar.md) |  
+| button | string | module uses a [button layout](../layouts/button.md) |  
```

Comment:
Thinking of something like this for the tables. It's still a little repetitive, but cleans things up a bit, and still follows form. Thoughts?

<img width="749" alt="screen shot 2016-06-09 at 9 52 07 am" src="https://cloud.githubusercontent.com/assets/5973633/15938415/db8c0cb2-2e27-11e6-84f8-68120cb2ac6a.png">